server/src/application/application.errors.ts

```diff
@@ -0,0 +1,16 @@
+import { HttpStatus } from '@nestjs/common';
+import { GenericError } from '../common/generic-exception';
+
+export const ApplicationError = {
+  APPLICATION_NOT_FOUND: {
+    errorType: 'APPLICATION_NOT_FOUND',
+    errorMessage: 'Cannot find application',
+    httpStatus: HttpStatus.NOT_FOUND,
+  } as GenericError,
+
+  USER_EMPTY: {
+    errorType: 'USER_EMPTY',
```

Comment:
User not found

Reply:
This one is not used anywhere anymore, so removing